lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeNames.cpp

```diff
@@ -302,6 +302,7 @@ class ThreadPlanRunToAddressOnAsyncCtx : public ThreadPlan {
     auto &target = thread.GetProcess()->GetTarget();
     m_funclet_bp = target.CreateBreakpoint(destination_addr, true, false);
     m_funclet_bp->SetBreakpointKind("async-run-to-funclet");
+    m_funclet_bp->SetThreadID(thread.GetID());
```

Comment:
to refresh reviewer's memories: this is the thread plan to "run through a task switch and stop when we schedule this same ***async function*** again. This plan now show set thread specific breakpoints, since they are now task breakpoints!

Reply:
Should be this an inline comment in the source file ?

Reply:
I would argue that in this case we don't need any comments, because this is just the "natural" thing to do. If anything, it is the _old_ implementation that deserved a comment, as it was essentially making all threads stop.

Reply:
> This plan now sets thread specific breakpoints, since they are now task breakpoints!

I'm not sure I understand what you're saying with this.

Reply:
I just noticed that the writing was indeed confusing, my bad.
The important thing to note here is that this plan is no longer special, it should work like any other plan: when it sets a breakpoint, it sets a thread specific breakpoint (otherwise it would cause all threads to stop).

Reply:
A follow up patch will remove this plan in favor of the RunToBreakpointPlan